### PR TITLE
Calculator functionality

### PR DESCRIPTION
**Pull request by BQuinteros**
-added basic functionality to the calculator, you can add, substract, divide two operands
-added the design of the button clear
ezgif com-video-to-gif

### CODE REVIEW

File: app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java [hunk 1]
```diff
@@ -24,5 +30,11 @@ public void showResult(float result) {
         visorResult.setText(String.valueOf(result));
     }
 
+    public void showMessageOperation(){
+        Toast.makeText( getContext(), TOAST_ERROR_OPERATOR_MSG, Toast.LENGTH_SHORT).show();
```
Comment (SartiMau) [inline app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java]: Put the Toast message into a resource pls

File: app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java [hunk 2]
```diff
@@ -24,5 +30,11 @@ public void showResult(float result) {
...
+    }
 
+    public void showMessageDivide(){
+        Toast.makeText( getContext(), TOAST_MSG_DIVIDE, Toast.LENGTH_SHORT).show();
```
Comment (SartiMau) [inline app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java]: sane

File: app/src/main/java/com/example/calculadoramvp/utils/Utils.java [hunk 3]
```diff
@@ -2,9 +2,20 @@
 
 public class Utils {
     public static final String EMPTY_STRING = "";
-    public static final String TOAST_MSG_FIRST_OPERAND = "Please insert the first operand";
-    public static final String TOAST_ERROR_MSG = "Please insert an operator";
+    public static final String TOAST_ERROR_OPERATOR_MSG = "Please insert an operator";
```
Comment (SartiMau) [inline app/src/main/java/com/example/calculadoramvp/utils/Utils.java]: remove the toast messages from here pls

**Review by SartiMau — CHANGES_REQUESTED** [review]

**Review by lucianoeli — APPROVED** [review]

File: app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java (hunk 1 above)
Comment (BQuinteros) [reply app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java]: done.

File: app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java (hunk 2 above)
Comment (BQuinteros) [reply app/src/main/java/com/example/calculadoramvp/mvp/view/CalculatorView.java]: done

File: app/src/main/java/com/example/calculadoramvp/utils/Utils.java (hunk 3 above)
Comment (BQuinteros) [reply app/src/main/java/com/example/calculadoramvp/utils/Utils.java]: done

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**